### ISSUE TITLE
Update to Swift 5.8

### ISSUE THREAD

**Issue by xilopaint** [roeybiran/dash-docset-swift-5.3-language-guide#2]
Would it be possible?

**Comment by roeybiran** [comment]
Of course, but it will be a while as I’m busy with my day job and other stuff. Stay tuned!

**Comment by roeybiran** [comment]
Well, apparently updating the docset for Swift 5.8/5.9 would be harder than I thought. The Swift docs are now built using  DocC, essentially a JS-based single page application and that doesn’t seem to play well with Dash.
